internal/tests/api/main_test.go

```diff
@@ -20,6 +20,16 @@ var (
 	hdNode  *hdtesting.HyperdriveNode
 )
 
+var baseSnapshot string
+
+func TestMain_BaseSnapshot(t *testing.T) {
```

Comment:
See previous comment - this should be done as part of `TestMain` instead of as its own unit test, so the base snapshot is guaranteed to exist.